packages/playground/Samples/viewKeyBoardEvents.tsx

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
We should have an RNTester test page for keyboard events already I think

Reply:
I see this one for Win32 https://github.com/microsoft/react-native-windows/blob/master/packages/@office-iss/react-native-win32/src/Libraries/Components/View/Tests/ViewWin32Test.tsx

But I'm having trouble spotting one for uwp.

Either way I'm happy to remove this if it isn't a valuable addition. I just thought it would make the PR more complete to include a demo of what breaks without the fix.